### ISSUE TITLE
Pulumi DigitalOcean plugin is not installed correctly on Mac 

### ISSUE THREAD

**Issue by Peddle** [pulumi/pulumi-digitalocean#14]
System: macOS Mojave 10.14.5

I get the following error when I try to run `pulumi up`
```
error: failed to load plugin /Users/aaronpeddle/.pulumi/plugins/resource-digitalocean-v0.18.2/pulumi-resource-digitalocean: fork/exec /Users/aaronpeddle/.pulumi/plugins/resource-digitalocean-v0.18.2/pulumi-resource-digitalocean: no such file or directory
```

An ls of that directory shows the following file:
```-rwxr-xr-x  1 xxxxxxx  xxxx  40972512 Jun 15 19:13 pulumi-resource-digitalocean{BIN_SUFFIX}```

I renamed the file to be `pulumi-resource-digitalocean` and it works.

I apologize if I'm posting this issue under the wrong repo. I'm still trying to figure out pulumi and I could find almost zero reference to a digitalocean plugin anywhere in the docs

**Comment by benbjurstrom** [comment]
Having the same issue in MacOS and in an Ubuntu ci runner. Adding this to my build script got things working again.

`mv $HOME/.pulumi/plugins/resource-digitalocean-v0.18.2/pulumi-resource-digitalocean\{BIN_SUFFIX\} $HOME/.pulumi/plugins/resource-digitalocean-v0.18.2/pulumi-resource-digitalocean`

**Comment by stack72** [comment]
@Peddle / @benbjurstrom 	

My apologies for this issue - it's something we should have caught before it was even published. I just opened a PR for a fix to this

I will post here when 0.18.3 has been released

Paul

**Comment by stack72** [comment]
Hi both

I have just released 0.18.3 of the provider and can confirm that it works as expected:

```
▶ pulumi up
Please choose a stack, or create a new one: dev-2
Previewing update (dev-2):

     Type                          Name                      Plan
 +   pulumi:pulumi:Stack           do-install-testing-dev-2  create
 +   └─ digitalocean:index:Domain  pulumi-test.com           create

Resources:
    + 2 to create

Do you want to perform this update? yes
Updating (dev-2):

     Type                          Name                      Status
 +   pulumi:pulumi:Stack           do-install-testing-dev-2  created
 +   └─ digitalocean:index:Domain  pulumi-test.com           created

Outputs:
    name: "pulumi-test.com-21e1dee"

Resources:
    + 2 created

Duration: 5s
```

Apologies for the problems again

Paul
